dockstore/dockstore
# Harvest DOIs from Zenodo

**Description**

The code looks for Zenodo DOIs issued by the Zenodo-GitHub integration and assigns them to the corresponding Dockstore workflows. Here, workflow refers to the Dockstore Workflow class, which means it includes BioWorkflows, Notebooks, AppTools, and Services.

If multiple workflows exist in a single GitHub repo, the DOI(s) are assigned to all workflows, a decision reached at a story time.

**Review Instructions**
1. Navigate to the nf-core/rnavar workflow in Dockstore, verifying there are no DOIs associated with it.
2. Invoke the endpoint `/workflows/updateDois` with the filter parameter set to `nf-core/rnavar`. Admin privileges required.
3. Navigate to the nf-core/rnavar workflow in Dockstore again, and verify that there are DOIs associated with it.

**Issue**
dockstore/dockstore#5745
DOCK-2493

**Security and Privacy**

If there are any concerns that require extra attention from the security team, highlight them here and check the box when complete. 


e.g. Does this change...
* Any user data we collect, or data location?
* Access control, authentication or authorization?
* Encryption features?

Please make sure that you've checked the following before submitting your pull request. Thanks!
